engine/runner/src/main/java/org/enso/runner/Main.java

```diff
@@ -1267,13 +1272,13 @@ private void launch(String[] args) throws IOException, InterruptedException, URI
             commandAndArgs.add(op);
           }
         }
-
+        var assertsOn = false;
+        assert assertsOn = true;
+        if (assertsOn) {
```

Comment:
This _"transfer of assertion mode"_ is now causing [following failure](https://github.com/enso-org/enso/actions/runs/10347694029/job/28641119556?pr=10779), @Akirathan:
```
Execution finished with an error: java.lang.AssertionError
         at <java> org.enso.runtime/org.enso.interpreter.runtime.warning.WithWarnings.<init>(WithWarnings.java:82)
         at <java> org.enso.runtime/org.enso.interpreter.runtime.warning.AppendWarningNode.doObjectMultipleWarningsHashMap(AppendWarningNode.java:120)
         at <java> org.enso.runtime/org.enso.interpreter.runtime.warning.AppendWarningNodeGen.executeAndSpecialize(AppendWarningNodeGen.java:318)
         at <java> org.enso.runtime/org.enso.interpreter.runtime.warning.AppendWarningNodeGen.executeAppend(AppendWarningNodeGen.java:193)
         at <java> org.enso.runtime/org.enso.interpreter.runtime.data.atom.InstantiateNode.doExecute(InstantiateNode.java:102)
         at <java> org.enso.runtime/org.enso.interpreter.runtime.data.atom.InstantiateNodeGen.executeAndSpecialize(InstantiateNodeGen.java:95)
         at <java> org.enso.runtime/org.enso.interpreter.runtime.data.atom.InstantiateNodeGen.executeGeneric(InstantiateNodeGen.java:76)
         at <java> org.enso.runtime/org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:54)
         at <java> org.enso.runtime/org.enso.interpreter.node.ClosureRootNode.execute(ClosureRootNode.java:85)
         at <enso> Data.Value(Internal)
         at <enso> Data.type.new(src/Decimal_Benchmark.enso:30:9-180)
         at <enso> Decimal_Benchmark.collect_benches.Decimal_Benchmark.collect_benches<arg-1>(src/Decimal_Benchmark.enso:37:29-42)
```
the 5575d8815a change should fix the problem.

Reply:
Alas, that fix leads to [this error](https://github.com/enso-org/enso/actions/runs/10383327368/job/28748030705#step:7:3843):
```
️ Execution finished with an error: java.lang.AssertionError
 ️at <java> org.enso.interpreter.runtime.warning.WithWarnings.<init>(WithWarnings.java:84)
 ️at <java> org.enso.interpreter.runtime.warning.AppendWarningNode.doWithWarnMultipleWarningsHashMap(AppendWarningNode.java:140)
  at <java> org.enso.interpreter.runtime.warning.AppendWarningNodeGen.executeAndSpecialize(AppendWarningNodeGen.java:345)
  at <java> org.enso.interpreter.runtime.warning.AppendWarningNodeGen.executeAppend(AppendWarningNodeGen.java:193)
  at <java> org.enso.interpreter.node.callable.InvokeMethodNode.doWarning(InvokeMethodNode.java:479)
 ️at <java> org.enso.interpreter.node.callable.InvokeMethodNodeGen.executeAndSpecialize(InvokeMethodNodeGen.java:781)
 ️at <java> org.enso.interpreter.node.callable.InvokeMethodNodeGen.execute(InvokeMethodNodeGen.java:533)
 ```
let's assume 0b9a49b fixes both - the instantiate and the invoke case.